src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -415,7 +415,7 @@ class LicenceService(
     try {
       val matchingLicences =
         licenceRepository.findAll(licenceQueryObject.toSpecification(), licenceQueryObject.getSort())
-      return matchingLicences.map { it.toSummary() }
+      return matchingLicences.filterNot { it.kind == LicenceKind.HDC }.map { it.toSummary() }
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of. -->
**note:** ‏This is to currently exclude any HDC licence that may be created from surfacing to the caseload which use the `match` endpoint

Reply:
**suggestion**: think it's fine to not include this... it will hopefully reveal extra things we need to consider